### PR TITLE
q-dir: Persist start.qdr

### PR DESCRIPTION
**Pull request by RustamGamidov**
Opened tabs are stored in the file not listed in the persistent section.
Q-DIr is happy with the empty _start.qdr_ and at it's updated at the first start

### CODE REVIEW

File: bucket/q-dir.json [hunk 1]
```diff
@@ -38,12 +38,16 @@
...
+        "Q-Dir.ini",
+        "start.qdr"
     ],
     "checkver": {
         "url": "https://www.softwareok.com/?seite=Microsoft/Q-Dir/History",
         "regex": "version ([\\d.]+)"
     },
+    "pre_install": [
+        "$null >> \"$persist_dir\\start.qdr\""
+    ],
```
Comment (Ash258) [inline bucket/q-dir.json:50]:
```suggestion
```

File: bucket/q-dir.json [hunk 2]
```diff
@@ -38,12 +38,16 @@
     "extract_dir": "Q-Dir",
     "persist": [
         "Favoriten",
-        "Q-Dir.ini"
+        "Q-Dir.ini",
+        "start.qdr"
     ],
```
Comment (Ash258) [inline bucket/q-dir.json:43]:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\start.qdr\")) { New-Item \"$dir\\start.qdr\" | Out-Null }",
    "persist": [
        "Favoriten",
        "start.qdr",
        "Q-Dir.ini"
    ],
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/q-dir.json (hunk 2 above)
Comment (RustamGamidov) [reply bucket/q-dir.json:43]: shouldn't it be `$persist_dir` in the `New-Item \"$dir\\start.qdr\"` ?

Comment (Ash258) [reply bucket/q-dir.json:43]: No.

**Pull request merged**